DESCRIPTION

```diff
@@ -70,6 +70,7 @@ Imports:
     tangram,
     tikzDevice,
     tinytex,
+    trackdown,
```

Comment:
I think you also need to add the `Remotes:` field in DESCRIPTION for it. This is the only package we install that is from Github I think.

This is to be perfectly complete, even is the lockfile for renv already contain the information

Reply:
Awesome thanks ! 
9c8e8f4

Reply:
Thanks! I added the required information to the `DESCRIPTION` file in commit https://github.com/yihui/rmarkdown-cookbook/pull/348/commits/9c8e8f43630ded0ad81394e46435831311dce5c1